### DISCUSSION TITLE
can argo rollout analysis template support remote prometheus/grafana cloud

### DISCUSSION THREAD

**Discussion by Hardroc** [argoproj/argo-rollouts#2433] [Q&A]
Hi All,
can anyone clarify whether argo rollouts analysis template support remote rewrite prometheus/grafana cloud ?
we have a use case and all workloads are running in GKE and grafana agent is running locally pushes the metrics to grafana cloud, when we use the same address for analysis run it is failing with 404 error

**Comment by dromadaire54** [comment]
Hi everybody,
I have an issue with an argo rollout s analysis template the provider prometheus gives me an 401 error when I want to query the prometheus instance I already add an API token with read scope.
Regards
